CHANGES.md

```diff
@@ -1,3 +1,139 @@
+## Version 1.0.0-MF
+
+> 2017 June 3
+
+`MF` stands for milestone final. This is the last non-RC release before 1.0.0.
+ The main purpose/focus of this release is to offer a relatively stable API to
+ work with prior to 1.0.0. It can be deemed as a proposal for the final API
+ we are going to maintain binary compatibiliy after 1.0. 
+ We will give community some time to validate it before we release 1.0.0-RC1.
+ 
+To migrate from 0.9.0
+
+ * `cats.free.Inject` is moved from `cats.free` to `cats.core` and renamed to `cats.InjectK`;
```

Comment:
this implies moving to a `cats.core` package-- maybe use `cats-core` and `cats-free` to refer to modules?